src/inatinqperf/utils/dataio.py

```diff
@@ -0,0 +1,41 @@
+"""Utilities for data I/O operations."""
+
+import csv
+import numpy as np
+from datasets import load_dataset, concatenate_datasets, Dataset
+from PIL import Image
+from pathlib import Path
+
+
+def load_composite(hf_id: str, expr: str) -> Dataset:
```

Comment:
Is there any reason that `expr` is a string rather than an array of strings? It looks like arguments for this parameter are generally read from a configuration file, which can also be an array of strings.